explorer/nodewatch/nodewatch/__init__.py

```diff
@@ -14,14 +17,27 @@ def create_app():
 	app.config.from_envvar('NODEWATCH_SETTINGS')
 
 	resources_path = Path(app.config.get('RESOURCES_PATH'))
-	min_cluster_size = app.config.get('MIN_HEIGHT_CLUSTER_SIZE', MIN_HEIGHT_CLUSTER_SIZE)
 	network_name = app.config.get('NETWORK_NAME', 'mainnet')
+	min_cluster_size = app.config.get('MIN_HEIGHT_CLUSTER_SIZE', MIN_HEIGHT_CLUSTER_SIZE)
+
+	symbol_explorer_endpoint = app.config.get('SYMBOL_EXPLORER_ENDPOINT')
+	symbol_generation_hash_seed = app.config.get('SYMBOL_GENERATION_HASH_SEED', None)
+	nem_explorer_endpoint = app.config.get('NEM_EXPLORER_ENDPOINT')
+
 	log.info(f'loading resources from {resources_path}')
-	log.info(f' configured with MIN_HEIGHT_CLUSTER_SIZE {min_cluster_size}')
 	log.info(f' configured with NETWORK_NAME {network_name}')
+	log.info(f' configured with MIN_HEIGHT_CLUSTER_SIZE {min_cluster_size}')
+	log.info(f' configured with SYMBOL_EXPLORER_ENDPOINT {symbol_explorer_endpoint}')
+	log.info(f' configured with SYMBOL_GENERATION_HASH_SEED {symbol_generation_hash_seed}')
+	log.info(f' configured with NEM_EXPLORER_ENDPOINT {nem_explorer_endpoint}')
+
+	nem_network = NetworkLocator.find_by_name(NemNetwork.NETWORKS, network_name)
+	symbol_network = NetworkLocator.find_by_name(SymbolNetwork.NETWORKS, network_name)
+	if symbol_generation_hash_seed:
+		symbol_network.generation_hash_seed = symbol_generation_hash_seed
```

Comment:
The generation hash needs to be converted to Hash256.

Reply:
https://github.com/symbol/product/blob/48243dd6206c3396b7b07bdfb6b69b01b7a24eec/explorer/nodewatch/nodewatch/NetworkRepository.py#L185